.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+on:
+  push:
```

Comment:
are the trailing colons necessary here?

Reply:
Trailing colon is required, since it's a mapping, not a simple list/array. [Docs explicitly say so](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration)

> You must append a colon (:) to all events, including events without configuration